src/main/java/nl/tudelft/ti2206/group9/shop/skin/AbstractSkin.java

```diff
@@ -0,0 +1,61 @@
+package nl.tudelft.ti2206.group9.shop.skin;
+
+import javafx.scene.paint.PhongMaterial;
+import nl.tudelft.ti2206.group9.gui.Style;
+import nl.tudelft.ti2206.group9.shop.ShopItem;
+
+/**
+ * The abstract shop item class for skins, based on the requirements from the
+ * ShopItem interface. Besides, every skin item has it's own texture.
+ * @author Maikel and Mitchell.
+ */
+@SuppressWarnings("restriction") // SuppresWarnings, because
+// PhongMaterials are needed for the textures.
+public abstract class AbstractSkin implements ShopItem {
```

Comment:
AbstractSkin and AbstractSoundrack are indeed abstract. The SAT are fine with this, indeed.
Besides, so am I. :) The main reason, indeed, is that these classes shouldn't be instantiated.
Thanks for noticing anyway. :)

<review,logic